### PR TITLE
Fix lighbox when gallery has no siblings

### PR DESCRIPTION
**Pull request by leoxiong**
When a gallery has no siblings (i.e. a page containing only a gallery, or a gallery at the very start/end of a page), clicking to view a gallery image would throw an exception.

See MDN for nextSibling vs nextElementSibling: https://developer.mozilla.org/en-US/docs/Web/API/Node/nextSibling

### CODE REVIEW

File: packages/_shared/assets/js/lib/lightbox.js [hunk 1]
```diff
@@ -5,25 +5,26 @@ function lightbox(trigger) {
...
-                    h: item.getAttribute('height'),
-                    el: item,
-                })
-
-                index += 1;
-            });
-            prevSibling = prevSibling.previousSibling;
+                    index += 1;
+                });
+                prevSibling = prevSibling.previousSibling;
```
Comment (minimaluminium) [inline packages/_shared/assets/js/lib/lightbox.js:24]: Should be replaced with `previousElementSibling` here.

File: packages/_shared/assets/js/lib/lightbox.js [hunk 2]
```diff
@@ -54,19 +55,20 @@ function lightbox(trigger) {
...
+                nextSibling.querySelectorAll('img').forEach(function (item) {
+                    items.push({
+                        src: item.getAttribute('src'),
+                        msrc: item.getAttribute('src'),
+                        w: item.getAttribute('width'),
+                        h: item.getAttribute('height'),
+                        el: item,
+                    })
+                });
+                nextSibling = nextSibling.nextSibling;
```
Comment (minimaluminium) [inline packages/_shared/assets/js/lib/lightbox.js:70]: This one should also be `nextElementSibling`.

**Review by minimaluminium — CHANGES_REQUESTED** [review]
Hey there, nice find, thanks for that! There're a few things to be updated, and should be good to merge 🙂

**Comment by leoxiong** [comment]
Hey @minimaluminium updated :). Also added the built JS files.

**Comment by minimaluminium** [comment]
Hey @leoxiong, happy new year! Sorry, I meant changing `prevSibling = prevSibling.previousSibling;` to `prevSibling = prevSibling.previousElementSibling;`, and same for `nextSibling`. The variable names shouldn't be changed :)

**Comment by minimaluminium** [comment]
@leoxiong I've just closed this one, because it got bit harder to manage. Instead I've fixed the issue in https://github.com/TryGhost/Themes/commit/104133d0fa7ec75ccd2eefba4488a074f8cd74b9, and gave you a credit in the commit. Thank you 🙌

**Comment by leoxiong** [comment]
No worries, thanks.

**Pull request closed without merging**